front/pages/api/v1/apps/[user]/[sId]/runs/index.ts

```diff
@@ -126,13 +91,13 @@ export default async function handler(req, res) {
   let [app, providers] = await Promise.all([
     App.findOne({
       where: {
-        userId: reqUser.id,
+        userId: appOwner.id,
         sId: req.query.sId,
       },
     }),
     Provider.findAll({
       where: {
-        userId: reqUser.id,
+        userId: appOwner.id,
```

Comment:
providers should always come from authUser (even if they are the same here)

Reply:
Got it now 👍 But I was blindly replacing.